src/main/java/com/questhelper/quests/thedepthsofdespair/TheDepthsOfDespair.java

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright (c) 2021, Zoinkwiz
```

Comment:
I didn't know @Zoinkwiz wrote this!

Reply:
Good catch, forgot to fix that after the copy-paste. Fixed.